test/sagemaker_tests/huggingface/inference/makefile

```diff
@@ -0,0 +1,59 @@
+# Pytorch Tests
+
+
```

Comment:
How is this file used?

Reply:
This can be deleted after the tests are merge-ready.